server/test/controllers/ti/TrustedIntermediaryControllerTest.java

```diff
@@ -248,7 +248,6 @@ public void addClient_WithMissingDob() {
                 profileUtils.currentUserProfile(requestBuilder.build()).get())
             .get();
     Result result = tiController.addClient(trustedIntermediaryGroup.id, requestBuilder.build());
-    System.out.println(contentAsString(result));
```

Comment:
This was an erroneous print statement I found that I'm pretty sure was checked in by accident so I'm removing it.